test/specs/index.js

```diff
@@ -35,27 +37,74 @@ describe('lightbox', function() {
     expect($(BLOCKING_CLASS).is(':off-screen')).not.toBeTruthy();
   });
 
-  it('should go to the next image when clicking on the next button', function(done) {
-    this.init();
-    $(LIGHTBOX_CLASS).first().click();
-    tempWait(() => {
-      const $next = $(NEXT_CLASS);
-      const img = (id) => `img[src$="${id}.png"]`;
+  describe('previous/next buttons', function() {
+    beforeEach(function(done) {
+      this.init();
+      $(LIGHTBOX_CLASS).first().click();
+      tempWait(() => {
+        const $next = $(NEXT_CLASS);
 
-      expect($next).toBeVisible();
-      expect($(img(1))).toBeVisible();
-      expect($(img(2)).is(':off-screen')).toBeTruthy();
+        expect($next).toBeVisible();
+        expect($(img(1)).is(':off-screen')).toBeFalsy();
+        expect($(img(2)).is(':off-screen')).toBeTruthy();
+        expect($(img(1)).parent().css('opacity')).toBe('1');
+        expect($(img(2)).parent().css('opacity')).toBe('0');
 
-      $next.click();
+        $next.click();
 
+        tempWait(done);
+      });
+    });
+
+    it('should go to the next image when clicking on the next button', function() {
+      expect($(img(1)).parent().css('opacity')).toBe('0');
+      expect($(img(2)).parent().css('opacity')).toBe('1');
+      expect($(img(2)).is(':off-screen')).toBeFalsy();
+    });
+
+    it('should go to the previous image when clicking on the previous button', function(done) {
+      const $previous = $(PREVIOUS_CLASS);
+      $previous.click();
       tempWait(() => {
-        expect($(img(1)).parent().css('opacity')).toBe('0');
-        expect($(img(2))).toBeVisible();
```

Comment:
This line doesn't actually test what we need it to - it technically always returns `true`